spec/unit/pops/parser/lexer2_spec.rb

```diff
@@ -91,12 +91,25 @@ def epp_tokens_scanned_from(s)
     "true"     => :BOOLEAN,
     "in"       => :IN,
     "unless"   => :UNLESS,
+    "private"  => :PRIVATE,
+    "type"     => :TYPE,
+    "attr"     => :ATTR,
```

Comment:
They were just missing in the list of keywords being tested. It is unrelated. I forgot to make a comment to that effect.